### PR TITLE
add router

### PR DESCRIPTION
**Pull request by perlo27**
add loadable

### CODE REVIEW

File: src/utils/loadable.tsx [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import React, {ComponentType, lazy, Suspense} from 'react';
+
+export function loadable<PropsType> (
```
Comment (jwfwessels) [inline src/utils/loadable.tsx]: Should this not also be a named variable with arrow function like everything else for consistency?

File: src/utils/tests.tsx [hunk 2]
```diff
@@ -0,0 +1,29 @@
+import React from 'react';
```
Comment (jwfwessels) [inline src/utils/tests.tsx]: pinged you on slack about file name and purpose

File: src/components/Home/index.tsx [hunk 3]
```diff
@@ -1,3 +1,10 @@
 import React from 'react';
+import { Link } from 'react-router-dom';
 
-export const Home: React.FC = () => <div data-testid="home">HOME PAGE</div>;
+export const Home: React.FC = () => (
```
Comment (jwfwessels) [inline src/components/Home/index.tsx]: `React.FC` this is still super weird to me 😅

File: src/utils/loadable.tsx (hunk 1 above)
Comment (perlo27) [reply src/utils/loadable.tsx]: workaround to make typescript work with arrow-functions with generics looks ugly, 
left like this for now

**Pull request merged**